contracts/payloads/IGP97/PayloadIGP97.sol

```diff
@@ -0,0 +1,365 @@
+                });
+                setDexLimits(DEX_sUSDe_GHO); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(sUSDe_GHO_DEX, TEAM_MULTISIG, false);
+            }
+        }
+
+        // sUSDe-GHO / USDC-GHO
+        {
+            address USDC_GHO_DEX_ADDRESS = getDexAddress(4);
+            address sUSDe_GHO__USDC_GHO_VAULT_ADDRESS = getVaultAddress(125);
+
+            {
+                // Update sUSDe-GHO<>USDC-GHO vault borrow shares limit
+                IFluidAdminDex.UserBorrowConfig[]
+                    memory config_ = new IFluidAdminDex.UserBorrowConfig[](1);
+                config_[0] = IFluidAdminDex.UserBorrowConfig({
+                    user: sUSDe_GHO__USDC_GHO_VAULT_ADDRESS,
+                    expandPercent: 30 * 1e2, // 30%
+                    expandDuration: 6 hours, // 6 hours
+                    baseDebtCeiling: 4_000_000 * 1e18, // 4M shares ($8M)
+                    maxDebtCeiling: 5_000_000 * 1e18 // 5M shares ($10M)
+                });
+
+                IFluidDex(USDC_GHO_DEX_ADDRESS).updateUserBorrowConfigs(
+                    config_
+                );
+            }
+
+            VAULT_FACTORY.setVaultAuth(
+                sUSDe_GHO__USDC_GHO_VAULT_ADDRESS,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+    }
+
+    // @notice Action 2: Set dust limits for USDC-USDT-CONCENTRATED DEX and its Vaults
+    function action2() internal isActionSkippable(2) {
+        {
+            address USDC_USDT_CONCENTRATED_DEX = getDexAddress(34);
+            {
+                // USDC-USDT-CONCENTRATED DEX
+                {
+                    // USDC-USDT-CONCENTRATED Dex
+                    DexConfig memory DEX_USDC_USDT_CONCENTRATED = DexConfig({
+                        dex: USDC_USDT_CONCENTRATED_DEX,
+                        tokenA: USDC_ADDRESS,
+                        tokenB: USDT_ADDRESS,
+                        smartCollateral: false,
+                        smartDebt: true,
+                        baseWithdrawalLimitInUSD: 0, // $0
+                        baseBorrowLimitInUSD: 8_000, // $8k
+                        maxBorrowLimitInUSD: 10_000 // $10k
+                    });
+                    setDexLimits(DEX_USDC_USDT_CONCENTRATED); // Smart Debt
+
+                    DEX_FACTORY.setDexAuth(
+                        USDC_USDT_CONCENTRATED_DEX,
+                        TEAM_MULTISIG,
+                        true
+                    );
+                }
+            }
+        }
+        {
+            //sUSDe-USDT / USDT-USDC-CONCENTRATED
+            address sUSDe_USDT_DEX_ADDRESS = getDexAddress(15);
+            address USDT_USDC_CONCENTRATED_DEX_ADDRESS = getDexAddress(34);
+            address sUSDe_USDT__USDT_USDC_CONCENTRATED_VAULT_ADDRESS = getVaultAddress(
+                    126
+                );
+
+            {
+                // Update sUSDe-USDT<>USDT-USDC-CONCENTRATED vault borrow shares limit
```

Comment:
for both vaults we only need to set the vault supply side limits, for borrow side USDT-USDC-CONCENTRATED we can only set via MS after initialize

Reply:
ahh right